### ISSUE TITLE
CS-4.15: CentOS: Official New UI Missing in Packaging from Source 

### ISSUE THREAD

**Issue by efansoftware** [apache/cloudstack#4680]
##### ISSUE TYPE

 * Bug Report
May also:
 * Improvement Request
 * Documentation Report

##### COMPONENT NAME

~~~
UI - the official new UI as announced big feature
~~~

##### CLOUDSTACK VERSION

~~~
4.15.0.0
~~~

##### CONFIGURATION

N/A

##### OS / ENVIRONMENT

CentOS-7 in my trial

##### SUMMARY

Built packages from 4.15.0.0 source successfully, then use the generated rpms to upgrade an working 4.13.1.0 environment.
[cd packaging; ./package.sh -d centos7]
But no new UI load. Legacy UI works with nice warning though.
In the upgraded management server folder, I can see that old UI code is indeed put under "legacy" folder, left "index.html" and "error.html" two files only in the webapp.

Similar to but not exactly: https://github.com/apache/cloudstack/issues/4578

##### STEPS TO REPRODUCE

~~~
[cd packaging; ./package.sh -d centos7]
~~~

##### EXPECTED RESULTS

~~~

~~~

##### ACTUAL RESULTS

~~~

~~~

**Comment by shwstppr** [comment]
@efansoftware New UI has been added in 4.15 branch for packaging with https://github.com/apache/cloudstack/pull/4605
It will be available in 4.15.1 as cloudstack-ui package

**Comment by shwstppr** [comment]
This is not a bug, packaging code already contains cloudstack-ui package.
For 4.15.0, primate UI has to be built separately.
@efansoftware can you please check and close or comment.

**Comment by efansoftware** [comment]
Addressed in #4605
